src/domains/ecommerce-app/api/ecommerce-checkout/v1/_auth_request.xml.tpl

```diff
@@ -45,14 +45,14 @@
             </when>
             <otherwise>
                 <return-response>
-                    <set-status code="400"/>
+                    <set-status code="504"/>
```

Comment:
why is it a timeout in this case?

Reply:
I think the 4XX is correct as the input is not valid

Reply:
OK. So the task can be closed ?

Reply:
Or we can discuss it at a refinement

Reply:
Maybe a 422 is more suitable for this specific case

Reply:
can be a good idea! I report your idea on JIRA 🙂